podman_compose.py

```diff
@@ -669,6 +670,22 @@ def norm_ports(ports_in):
         ports_out.append(port)
     return ports_out
 
+def assert_network_exists(compose, net_name):
+    podman_version = sv.Version(compose.podman_version)
+    if sv.SimpleSpec('<3.1.0').match(podman_version):
+          output = compose.podman.output([], "network", ["ls"])
+          offset = None
+          for raw_line in output.splitlines():
```

Comment:
Output parsing should be moved to separate function and a unit tests added for it.